### PR TITLE
Seeff.BOSSDesktopv1 version 1.0.0

### PR DESCRIPTION
**Pull request by Seeff-DevHub**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/106766)

### CODE REVIEW

**Comment by Seeff-DevHub** [comment]
@microsoft-github-policy-service agree company="Microsoft"

**Comment by Seeff-DevHub** [comment]
@microsoft-github-policy-service agree company="Microsoft"

**Comment by Seeff-DevHub** [comment]
@microsoft-github-policy-service agree [company="{Seeff}"]

**Comment by Seeff-DevHub** [comment]
@microsoft-github-policy-service agree

**Comment by Seeff-DevHub** [comment]
@microsoft-github-policy-service agree company="Seeff"

**Comment by stephengillie** [comment]
Manual Validation ended with: 
image

**Comment by stephengillie** [comment]
With `Microsoft.DotNet.DesktopRuntime.7` preinstalled, the package installs and runs normally.

File: manifests/s/Seeff/BOSSDesktopv1/1.0.0/Seeff.BOSSDesktopv1.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: Seeff.BOSSDesktopv1
+PackageVersion: 1.0.0
```
[the remark this answers is not in the document]
Comment (Seeff-DevHub) [reply manifests/s/Seeff/BOSSDesktopv1/1.0.0/Seeff.BOSSDesktopv1.installer.yaml:5]: Thanks Stephen, Apologies this is my first submission and I am feeling my way through the process.

**Comment by ItzLevvie** [comment]
[Policy] Blocking-Issue

**Pull request closed without merging**